autotest/basic.cpp

```diff
@@ -0,0 +1,71 @@
+#include <QtTest>
+
+#include "qtkeychain/keychain.h"
+
+namespace
+{
+    QString serviceKey = QStringLiteral("QtKeychainTest");
```

Comment:
I wonder if this string should also contain a random component, so there can't be any clashes when running multiple builds/tests in parallel.
The risk would be leaving multiple "QtKeyChainTest-$randomHash" entries behind if the test repeatedly fails cleaning up, but I think that risk is small.

Yay for unit tests btw, finally time to set up github CI ;)

Reply:
I'll use a random key